EXILED_Main/Extensions/Player.cs

```diff
@@ -603,8 +603,12 @@ public static Room GetCurrentRoom(this ReferenceHub player)
 		/// </summary>
 		/// <param name="player"></param>
 		/// <param name="amount"></param>
-		public static void AddHealth( this ReferenceHub player, float amount ) =>
-			player.playerStats.health = Mathf.Clamp(amount, 1, player.playerStats.maxHP);
+		public static void AddHealth(this ReferenceHub player, float amount, bool capAtMax = false) {
```

Comment:
Won't changing signature break old plugins?

Reply:
AFAIK it shouldn't 🤔, it'll just set capAtMax to false by default, but could be wrong

Reply:
Still signature is changed, so plugins compiled with old version of EXILED will break